pages.pt_BR/ansible.md

```diff
@@ -0,0 +1,33 @@
+# ansible
+
+> Gerencia grupos de computadores remotamente utilizando SSH. (use o arquivo `/etc/ansible/hosts` para adicionar novos grupos/hosts).
+> Alguns subcomando como `ansible galaxy` possuis a própria documentação de uso.
+> Mais informações: <https://www.ansible.com/>.
+
+- Lista os hosts pertencentes a um grupo:
+
+`ansible {{group}} --list-hosts`
+
+- Realiza o ping de um grupo de hosts invocando o módulo ping:
+
+`ansible {{group}} -m ping`
+
+- Exibe fatos sobre um grupo de hosts invocando o módulo setup:
+
+`ansible {{group}} -m setup`
+
+- Executa um comando em um grupo de hosts invocando o módulo command com argumentos:
+
+`ansible {{group}} -m command -a '{{my_command}}'`
```

Comment:
Please translate the placeholders, like `inventory_file`, `my_commad`, etc., for get this easier for non-English speakers.

Reply:
I did the translation. I believe it should be correct now. Thanks for the review.